docs/quick-answers/linux/how-to-use-export/index.md

```diff
@@ -0,0 +1,145 @@
+---
+author:
+  name: Rajakavitha Kodhandapani
+  email: docs@linode.com
+description: 'This Quick Answer guide will explain how to use export.'
```

Comment:
Maybe expand this? "This Quick Answer guide will explain how to use the export command to view terminal variables, how to  assign variable values, and how to pass functions to other processes."

Also, we need an og_description.